README.md

```diff
@@ -125,11 +125,11 @@ This project is set up to use [jest](https://facebook.github.io/jest/) for tests
 
 ## End-to-End Tests
 
-This repository extends the end-to-end (E2E) tests found in the Mobile Gutenberg (MG) [project repository](https://github.com/WordPress/gutenberg). The majority of E2E tests should reside in the MG repository. However, experimental tests or non-core-related tests should be placed in the `gutenberg-mobile` repository.
+This repository extends the end-to-end (E2E) tests found in the [Gutenberg project repository](https://github.com/WordPress/gutenberg). The majority of E2E tests should reside in the `gutenberg` repository. However, experimental tests or non-core-related tests should be placed in the `gutenberg-mobile` repository.
 
-Reviewing the MG E2E test [documentation](https://github.com/WordPress/gutenberg/tree/trunk/packages/react-native-editor/__device-tests__#readme) is the best approach for understanding the test environment and how to set up your computer to run tests. Much of the information and approaches outlined there also apply to the tests found in this repository. The only difference being that this repository includes its own matching npm scripts to run the test, e.g. `npm run test:e2e:ios:local`.
+Reviewing the [Mobile Gutenberg E2E test documentation](https://github.com/WordPress/gutenberg/tree/trunk/packages/react-native-editor/__device-tests__#readme) is the best approach for understanding the test environment and how to set up your computer to run tests. Much of the information and approaches outlined there also apply to the tests found in this repository. The only difference being that this repository includes its own matching npm scripts to run the test, e.g. `npm run test:e2e:ios:local`.
 
-After reviewing the MG E2E test documentation, the following examples showcase running the E2E tests found in this repository using the repositories scripts.
+After reviewing the Mobile Gutenberg E2E test documentation, the following examples showcase running the E2E tests found in this repository using the repositories scripts.
```

Comment:
For this line in particular, this sentence intended to reference `gutenberg`, not `gutenberg-mobile`. So, `MG` should likely be replaced with `gutenberg` to match earlier proposed changes on line ~128. I'll update the PR to address this.